TUTORIAL.md

```diff
@@ -0,0 +1,245 @@
+#Guide to Solving and Reviewing Rails Blogs Nested Forms
+##Overview
+This lab will teach you how to implement nested forms. In this example, we will nest tags under posts, so a user can submit a post that has many tags.
+
+###Step 1: Set up your app
```

Comment:
Why did you know that was the first thing to do? Some simple sentence like "Let's see what the world looks like, To get going we need to do three things. 1) get all our gems in order (bundle) 2) setup the DB and 3)start the server"